scout/server/blueprints/variants/views.py

```diff
@@ -176,7 +176,8 @@ def variants(institute_id, case_name):
         severe_so_terms=SEVERE_SO_TERMS,
         cytobands=cytobands,
         page=page,
-        expand_search=str(request.method == "POST"),
+        expand_search=request.form.get("expand_search") != "False"
```

Comment:
I dunno, is there a reason to have it double negative? My brain would process it better as == "True". If it is for hidden None's perhaps add a default to the get?

Reply:
There are hidden Nones and "", I'll try to write it a bit more clear